returnn/datasets/meta.py

```diff
@@ -1390,7 +1390,7 @@ def __init__(
         seq_len_file,
         seq_tag_delim=";",
         remove_in_between_postfix=None,
-        repeat_in_between_last_frame_up_to_multiple_of=None,
+        force_align=None,
```

Comment:
Renaming/changing the purpose of this parameter is going to break existing setups that use this dataset. Usually we try to stay backwards compatible.

One solution here is to keep both parameters (and their defaults), and then assert that only one of them is set at a time and then apply the corresponding logic.

I'm also not sure `force_align` is a good name for this key, as AFAIK the term "forced alignment" is usually used for producing an alignment from a trained model. I'd rather go with something like `force_pad_truncate_data_to_target_length`.

Reply:
Thank you for your information, I did this because (to my best knowledge) nobody used this in training before:) But you are right, such potential issue should be avoided, I will add back the parameter before and change the name

Reply:
Note that I know that people did use `ConcatSeqsDataset` in training, even recently. :) (Although not sure if this specific option was used.)